components/layout/ContactNavigation.js

```diff
@@ -0,0 +1,37 @@
+import classes from "./AboutUsNavigation.module.css";
+import Link from "next/link";
+import { useRouter } from "next/router";
+
+function ContactNavigation() {
+
+    const router = useRouter();
+
+    return (
+        <nav className={classes.aboutUsNav}>
+            <ul className={classes.aboutUsMenu}>
+                <li>
+                    <Link href="/contact/contact">
+                        <a className={router.pathname == "/contact/contact" ? `${classes.active} ${classes.menuText}` : `${classes.menuText}`}>Kontakt</a>
```

Comment:
Try reading this as an expression. Remember to read code not like an adult reads books, but the way a 7-year-old reads books – one token at a time.

```tsx
router.pathname == "/contact/contact" ? `${classes.active} ${classes.menuText}` : `${classes.menuText}`
```

The expression consists of multiple sub-expressions, which in turn comprise sub-expressions again:

1. `router.pathname == "/contact/contact"`
  a. `router.pathname`
  b. `"/contact/contact"`
2. ``${classes.active} ${classes.menuText}``
  a. `classes.active`
  b. `classes.menuText`
3. ``${classes.menuText}``
  a. `classes.menuText`

Expression 2 builds a string using _template strings_, consisting of two other strings, each represented by variable expressions 2a and 2b. It will evaluate the variables, which have string values, and create a string that looks like `'first-value second-value'`. String templating is used here to join two strings using a space character ( ` `).

Expression 3 does the same thing, i.e. creates a template string that evaluates a variable and adds it to a string. But this time there is nothing else in the string, than the string represented by `classes.menuText`.

So Expression 3 is useless. You could use 3a directly. Like so:

```tsx
// Both versions, with the version I'd recommend being the second
router.pathname == "/contact/contact" ? `${classes.active} ${classes.menuText}` : `${classes.menuText}`
router.pathname == "/contact/contact" ? `${classes.active} ${classes.menuText}` : classes.menuText
```